### PR TITLE
Build on pre-configured killercoda's ubuntu playground (added documentation & Link to readme)

### PR DESCRIPTION
**Pull request by suhasgumma**
## Build on pre-configured killercoda's ubuntu playground
* [Pre-configured Killercoda's Ubuntu Playground](https://killercoda.com/suhas-gumma/scenario/kubescape-build-for-development)

* For people having trouble building in their local machines, killercoda's ubuntu playground will be extremely useful. But, as you know the playground lasts for only 60 minutes. Executing each command manually every hour is not a pleasant experience.  So, I've created this custom playground. The user just had to start the playground and wait for 5-6 minutes. 

## Pre-programmed actions executed by the playground
* Clone the official GitHub repository of `Kubescape`.
* Install cmake.
* Install `libgit2` dependency inside 'kubescape' directory.
* Execute `make build` inside 'kubescape' directory.
* The entire process involves executing multiple commands in order and it takes around 5-6 minutes to execute them all.

## Instructions to use the playground
* Apply changes you wish to make to the kubescape directory using text editors like `Vim`.
* Execute `make build` after the changes. (This will build the `kubescape` binary)
* Now, you can use Kubescape just like a normal user. Instead of using `kubescape`, use `./kubescape`. (Make sure you are inside kubescape directory because the command will execute the binary named `kubescape` in `kubescape directory`.

### CODE REVIEW

**Comment by suhasgumma** [comment]
Hi @slashben, Could you please review this?

**Comment by suhasgumma** [comment]
> Hi @slashben, Could you please review this?

If you want to preview the changed README.md file, visit this link:
https://github.com/suhasgumma/kubescape/blob/killercoda/README.md

**Review by slashben — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -359,6 +359,31 @@ View Kubescape scan results directly in [Lens IDE](https://k8slens.dev/) using k
...
+
+## Build on pre-configured killercoda's ubuntu playground
+
+* [Pre-configured Killercoda's Ubuntu Playground](https://killercoda.com/suhas-gumma/scenario/kubescape-build-for-development)
+
+<details><summary> Pre-programmed actions executed by the playground </summary>
+
+
+* Clone the official GitHub repository of `Kubescape`.
+* Install cmake.
```
Comment (dwertent) [inline README.md:371]: I think we should link here to the [Build on Linux](https://github.com/kubescape/kubescape#build-on-linuxmacos)

File: README.md [hunk 2]
```diff
@@ -359,6 +359,31 @@ View Kubescape scan results directly in [Lens IDE](https://k8slens.dev/) using k
...
+* Execute `make build` inside 'kubescape' directory.
+* The entire process involves executing multiple commands in order and it takes around 5-6 minutes to execute them all.
+
+</details>
+
+<details>
+<summary>Instructions to use the playground</summary>
+
+* Apply changes you wish to make to the kubescape directory using text editors like `Vim`.
+* Execute `make build` after the changes. (This will build the `kubescape` binary)
```
Comment (dwertent) [inline README.md:382]: Here again, we should have a [single place](https://github.com/kubescape/kubescape#build-on-linuxmacos) of instructions on how to build

**Review by dwertent — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (suhasgumma) [reply README.md]: Hi @dwertent , I have made the changes you suggested.

**Pull request closed without merging**